repose-aggregator/src/docs/asciidoc/recipes/quick-start.adoc

```diff
@@ -0,0 +1,166 @@
+
+== Running a Docker Container
+After you have a running Docker Container Engine, you can run the latest *Repose* release with the following command:
+
+----
+docker run                  \ <1>
+   --detach                 \ <2>
+   --publish 8080:8080      \ <3>
+   rackerlabs/repose:latest   <4>
+----
+Let's break that command down and take a closer look at what it is doing:
+
+<1> This is the part of the command that tells Docker to create and start a container.
+<2> This option will run the container in detached mode.
+    In detached mode, the container will run in the background.
+    If we were to run in attached mode (the default) instead of detached mode, the terminal we used to execute this command would receive all of the container output (e.g. STDOUT, and STDERR).
+<3> This option defines a port mapping between the host (i.e., the OS running the Docker Container Engine) and the Docker container itself.
+    Doing so allows *Repose* to accept traffic over the specified port from outside of Docker.
+    The port mapping format is `HOST_PORT:CONTAINER_PORT`.
+    So in this example, traffic on host port 8080 will be forwarded to the containers port 8080.
+<4> This is the image ID (in repository:tag format) to create a container from.
+    Remember that *Repose* images are hosted at Docker Hub, which Docker can use implicitly.
+
+The `docker run` command will return the full 64 character unique ID of the Container that was started.
+However, only the first 12 are typically needed for the `CONTAINER_ID` in the rest of the command examples.
+If you no longer have the Container ID or you just want to see your handy work up to this point, then you can run:
+
+----
+docker ps
+----
+
+== Inspecting a Running Docker Container
+Once running, you may wish to inspect the internal state of the container.
+Well, the default command for the official *Repose Docker* images will run *Repose* in the foreground, and as a result, simply attaching to the container with a command like the following will only allow interaction with the *Repose* process:
+
+----
+docker attach CONTAINER_ID
+----
+
+Typically that is not very usefull.
+Instead, you can use a command like the one below to execute a new bash session inside of your running container:
+
+----
+docker exec --interactive --tty CONTAINER_ID /bin/bash
+----
+
+In that shell you can check the current filter chain being used by the *Repose* instance with:
+
+----
+cat /etc/repose/system-model.cfg.xml
+----
+
+To end the interactive session, issue the `exit` command.
+
+== Docker Container Logging
+The expectations of https://12factor.net/logs[Twelve-Factor App logging] is that all internal logging mechanisms within the container be disabled.
+The intent is that all traditional logging information should be streamed on STDOUT/STDERR.
+To access a containers log information simply use the following:
+
+----
+docker container logs --follow CONTAINER_ID
```

Comment:
I think you can drop the `container` bit from this command. `docker logs` is the shorthand, and seems to be the preferred way of accessing logs.

Reply:
Cool. Updated.